src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -362,21 +373,32 @@ def producer() -> Producer:
     return _get().producer()
 
 
-def ev_charger_pool(ev_charger_ids: abc.Set[int] | None = None) -> EVChargerPool:
-    """Return the corresponding EVChargerPool instance for the given ids.
+def ev_charger_pool(
+    ev_charger_ids: abc.Set[int] | None = None,
+    name: str | None = None,
+    priority: int = -sys.maxsize - 1,
+) -> EVChargerPool:
+    """Return a new `EVChargerPool` instance for the given parameters.
 
-    If an EVChargerPool instance for the given ids doesn't exist, a new one is
-    created and returned.
+    The priority value is used to resolve conflicts when multiple actors are trying to
+    propose different power values for the same set of EV chargers.
+
+    !!! note
+        When specifying priority, bigger values indicate higher priority. The default
+        priority is the lowest possible value.
```

Comment:
What if the priority is equal? Which one will then be selected?

Reply:
If there are multiple proposals with the same priority, they will be treated as having different priorities based on their randomly generated `source_id`.  So the priority between actors will be stable and they won't get into a race with each other when they have the same priority values.

Reply:
Maybe add a short form or hint of htat to the docu here :)

Reply:
Will do as part of this issue: https://github.com/frequenz-floss/frequenz-sdk-python/issues/902

Reply:
A small side note, in docstrings you can use normal sections for admonitions:

```
Note:
    When specifying priority, bigger values indicate higher priority. The default ...
```

You only need the "special" syntax if you need some customization (like if you don't want it to be collapsible or to show it on the side for example).

Reply:
Done in #904